lib/app/app.dart

```diff
@@ -24,6 +23,10 @@ class JuntoApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MultiProvider(
       providers: <SingleChildWidget>[
+        ChangeNotifierProvider<UserDataProvider>(
+          create: (ctx) => backend.dataProvider,
```

Comment:
it shouldn't use create if it's using value

Reply:
Fixed